v2/samples/azureiotedge-simulated-temperature-sensor/Docker/linux-arm/Dockerfile.debug

```diff
@@ -0,0 +1,15 @@
+FROM microsoft/dotnet:2.0.0-runtime-stretch-arm32v7
```

Comment:
This is another place where you can pull from azureiotedge/azureiotedge-module-base:1.0.0-preview001-linux-arm32v7, also we recently upgraded to 2.0.5